src/abi.md

```diff
@@ -0,0 +1,9 @@
+# Application Binary Interface (ABI)
+
+This section documents (or will document) features that affect the ABI of a Rust program / binary,
```

Comment:
We avoid talking about future plans in the reference. Both the parentheticals here should be removed.